x-pack/test/functional_with_es_ssl/apps/cases/group1/sub_privileges.ts

```diff
@@ -119,6 +128,59 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         });
       }
     });
+
+    describe('cases_settings', () => {
+      beforeEach(async () => {
+        await cases.api.createNthRandomCases(2);
+      });
+
+      afterEach(async () => {
+        await cases.api.deleteAllCases();
+      });
+
+      for (const user of [casesReadSettingsUser, casesAllUser]) {
+        describe(`logging in with user ${user.username}`, () => {
+          before(async () => {
+            await PageObjects.security.login(user.username, user.password);
+          });
+
+          after(async () => {
+            await PageObjects.security.forceLogout();
+          });
+
+          it(`User ${user.username} can navigate to settings`, async () => {
+            await cases.navigation.navigateToConfigurationPage();
+          });
+
+          it(`User ${user.username} can update settings`, async () => {
+            await cases.common.selectRadioGroupValue(
+              'closure-options-radio-group',
+              'close-by-pushing'
+            );
+            const toast = await toasts.getElementByIndex(1);
+            expect(await toast.getVisibleText()).to.be('Settings successfully updated');
+            await toasts.dismissAll();
+          });
+        });
+      }
+
+      for (const user of [casesNoDeleteUser, casesReadDeleteUser]) {
```

Comment:
I'm confused by these, is this maybe a leftover from copying `describe('cases_delete', () => {`? Why would delete permissions affect the visibility of the settings button?

The relevant test seems to be in line 141 with `for (const user of [casesReadSettingsUser, casesAllUser])`.

Reply:
These 2 users do not have `settings: true` privilege. So checking that `Settings` button is not visible.

Reply:
I agree that we should test this scenario. Maybe we could add a comment on top of the users to say that the users do not have access to settings.